unpacker.go

```diff
@@ -245,6 +247,31 @@ EachLayer:
 		"chainID": chainID,
 	}).Debug("image unpacked")
 
+	if rCtx.Deduplicate {
+		// delete references to successfully unpacked layers
+		layersMap := map[string]struct{}{}
+		for _, desc := range layers {
+			layersMap[desc.Digest.String()] = struct{}{}
+		}
+		pinfo, err := cs.Info(ctx, parentDesc.Digest)
+		if err != nil {
+			return err
+		}
+		fields := []string{}
+		for k, v := range pinfo.Labels {
+			if strings.HasPrefix(k, "containerd.io/gc.ref.content.") {
```

Comment:
.0 is config :) maybe we can use `content.config` and `content.layer.x` to label it when fetch.

ref: https://github.com/containerd/containerd/issues/3552

Reply:
There seems a lot of discussions on the suffix. I kept this patch config-agnostic but please tell me if you have anything I should include in this patch :)

Reply:
Sure~. SGTM